be/src/olap/tablet_schema.cpp

```diff
@@ -1367,7 +1367,13 @@ const TabletIndex* TabletSchema::get_inverted_index(int32_t col_unique_id,
     return nullptr;
 }
 
-const TabletIndex* TabletSchema::get_inverted_index(const TabletColumn& col) const {
+const TabletIndex* TabletSchema::get_inverted_index(const TabletColumn& col,
+                                                    bool check_valid) const {
+    // With check_valid set to true by default
+    // Some columns from the variant do not support inverted index
+    if (check_valid && !segment_v2::InvertedIndexColumnWriter::check_column_valid(col)) {
```

Comment:
The name check_column_valid is not so good. Suggest a new one check_support_inverted_index.

Reply:
done